### DISCUSSION TITLE
solving pde using quantum differentiable circuits

### DISCUSSION THREAD

**Discussion by Ashs111** [QuantumBFS/Yao.jl#499] [Q&A]
how to build the quantum circuit

**Comment by Roger-luo** [comment]
we cannot help you if you don't provide more information about which algorithm this is and what PDE you are looking for. Also, note that we do not in general answer homework questions. For what is relevant please see the circuits defined in https://github.com/QuantumBFS/QuDiffEq.jl

**Reply by Ashs111** [reply]
I am trying to build a quantum circuit to solve differential equations. This is the paper I am referring to Kyriienko, Oleksandr, Annie E. Paine, and Vincent E. Elfving. "Solving nonlinear differential equations with differentiable quantum circuits." Physical Review A 103.5 (2021): 052416. I want to develop a basic framework to manipulate it to solve any differential equation. Is the Python code of the problem discussed in the above-mentioned paper available? Can anyone suggest resources to refer to and how to develop and build the quantum circuit?
